### PR TITLE
Site beta v1

### PR DESCRIPTION
**Pull request by imfunniee**
made the site more sexi

### CODE REVIEW

File: pages/api/upvotes.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import axios from "axios";
+import { load } from "cheerio";
+
+type Data = {
+  upvotes: number;
+};
+
+export default async (req: NextApiRequest, res: NextApiResponse<Data>) => {
+  try {
+    console.log("count");
```
Comment (looskie) [inline pages/api/upvotes.ts]:
```suggestion
```

File: pages/api/upvotes.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    const { data } = await axios.get(
+      "https://www.producthunt.com/products/capybara-api"
+    );
+    const $ = load(data);
+    const upvotes = Number(
+      $("div")
```
Comment (looskie) [inline pages/api/upvotes.ts:15]: Feels wrong and inefficient using cheerio with next API routes let alone react in general, is there no public API for this?

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (looskie) [inline .vscode/settings.json:1]: can we actually add .vscode to gitignore

File: utils/Consts.ts [hunk 4]
```diff
@@ -59,23 +59,9 @@ after Capybara api: :ChocolaPOGGERS:`,
...
-    pfp: "/img/shinku.png",
-    worksAt: "ur moms house",
-  },
-  {
-    message:
-      "did you know that capypabaras eat their own poop in the morning? unique diet. protein rich. he pull up.",
-    name: "Ari",
-    pfp: "/img/ari.png",
-    worksAt: "Buildergroop Corporation",
-  },
```
Comment (looskie) [inline utils/Consts.ts]: add these back

File: utils/Fetcher.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+import { Fetcher } from "swr";
+
+const _getData = async (arg: string) => {
+  const data = await axios.get(arg).then((res) => res.data);
+  return data.upvotes.toString();
+};
+
```
Comment (looskie) [inline utils/Fetcher.ts:8]: can we make this into one whole function, theres no point in a "private" function if its just going to be exported.

**Review by looskie — CHANGES_REQUESTED** [review]

**Comment by looskie** [comment]
overall prettyy good!

File: utils/Fetcher.ts (hunk 5 above)
Comment (imfunniee) [reply utils/Fetcher.ts:8]: wdym

File: utils/Consts.ts (hunk 4 above)
Comment (imfunniee) [reply utils/Consts.ts]: to keep one of each deleted the ones being repeated

File: pages/api/upvotes.ts (hunk 2 above)
Comment (imfunniee) [reply pages/api/upvotes.ts:15]: there is but it requires token and its graphql

Comment (looskie) [reply pages/api/upvotes.ts:15]: bruhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh

File: utils/Consts.ts (hunk 4 above)
Comment (looskie) [reply utils/Consts.ts]: ohh okay my b

File: utils/Fetcher.ts (hunk 5 above)
Comment (looskie) [reply utils/Fetcher.ts:8]: like we dont need two functions here, we can just

```ts
const fetcher: Fetcher<string> = async (url: string) => {
  const data = await axios.get(arg).then((res) => res.data);
  return data.upvotes.toString();
}

**Review by looskie — APPROVED** [review]
lgtm!!!!!! thank you!

**Comment by imfunniee** [comment]
anytime for teammate

**Comment by h3xjs** [comment]
abow

**Pull request merged**